### DISCUSSION TITLE
Backup Part-DB - error message

### DISCUSSION THREAD

**Discussion by rdalen** [Part-DB/Part-DB-server#597] [General]
Hi, I am running Part-DB in a docker container on my raspberry pi.
I can make a backup by running `docker exec --user=www-data partdb php bin/console partdb:backup --full backup-partdb.zip`
But then I find the backup file deep down in /var/lib/docker subfolder.
When I give a file path in the output parameter (e.g. `docker exec --user=www-data partdb php bin/console partdb:backup --full /home/pi/containers/part-db/backup/backup-partdb.zip)` then I get a error message;
```
In ZipFile.php line 1488:

  fopen(/home/pi/containers/part-db/backup/backup-partdb.zip.temp661eb68f58e
  8f): Failed to open stream: No such file or directory
```
Even when I add sudo before the backup command- Any idea why I get this?

**Comment by jbtronics** [comment]
The backup command runs inside the docker container and the docker container can not access the host system. You need to map a folder from the host system into the docker container (or use one of the existing volume mappings) and output the backup into the mapped folder.

**Reply by rdalen** [reply]
Of course!!! I now use the existing volume mapping `/var/www/html/uploads` and that works!
